### PR TITLE
Add hmaarrfk as maintainer

### PR DESCRIPTION
**Pull request by hmaarrfk**
@conda-forge-admin please rerender
cc: @isuruf

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@hmaarrfk, can you send a PR to update the version? I don't think we want to merge this as is.

**Comment by hmaarrfk** [comment]
Fernando, unfortunately my time today is rather limited.

Happy to let this hang around if you want me to.

Otherwise, i simply skipped all package generation to allow this to go through without generating a package.

**Comment by nehaljwani** [comment]
Upstream feedstock for reference: https://github.com/pytorch/vision/blob/635406c3325e276ec8d0055d296094e113b44b20/packaging/torchvision/meta.yaml

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,27 @@ source:
   sha256: 91f1e21121dca7e14ec35b5f2127da77c8c52e61d39957ab5dd4a719a837a04c
 
 build:
-  number: 1000
-  skip: True  # [not linux64]
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 1001
```
Comment (nehaljwani) [inline recipe/meta.yaml:13]: Why is this 1001 and not 0?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,21 +10,27 @@ source:
...
+    - {{ compiler('cuda') }}    # [cuda_compiler_version != "None"]
+  host:
     - python
     - setuptools
-
+    - numpy
+    - cudnn                           # [cuda_compiler_version != "None"]
+    - libpng
+    - jpeg
+    - ffmpeg =4.2  # [not win]
```
Comment (nehaljwani) [inline recipe/meta.yaml:27]: Should we put the pin in CBC.yaml instead?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% set version = "0.2.1" %}
+{% set version = "1.7.1 " %}
```
Comment (nehaljwani) [inline recipe/meta.yaml:1]: Shouldn't this be 0.8.2?

**Comment by h-vetinari** [comment]
Just saw this PR, sorry, I overlooked it when opening #12. I think I'm a bit further there already though - the package builds and imports fine, just the test suite that I added to the test section is giving me some grief related to `jpeg` integration.

I'll send you an invite to collaborate on my fork @hmaarrfk, then you can push to #12 directly. There's still some open TODOs, see OP, but most of them can be tackled in follow-ups, IMO.

**Comment by hmaarrfk** [comment]
thanks!

**Pull request closed without merging**